.travis.yml

```diff
@@ -21,6 +21,7 @@ install:
 
 # running tests
 script:
+  - cd /tmp
   - pytest -v --maxfail=2 --cov=jupyterhub jupyterhub/tests
```

Comment:
Path of `jupyterhub/tests` needs to be updated.

Reply:
Remove